### ISSUE TITLE
NTR: Poliovirus RT-PCR in stool or oropharynx positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8075]
New term request Poliovirus RT-PCR in stool or oropharynx positive (samant_110207040554): Reverse transcriptase polymerase chain reaction; RNA  UMLS: 2748279

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000580
- label: Positive oropharynx poliovirus nucleic acid test
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: Detection of poliovirus nucleic acid in the oropharynx by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8075
- synonyms: 


PS, please open new request if additional term for stool needed.
